packages/smooth_app/lib/pages/product/common/country_wiki_links.dart

```diff
@@ -0,0 +1,25 @@
+class CountryWikiLinks {
+
+
+final Map<String, String> wikiLinks = <String, String>{
+      'Argentina': 'https://wiki.openfoodfacts.org/Country_Support_-_Argentina',
+      'Australia':'https://wiki.openfoodfacts.org/Country_Support_-_Australia',
+      'Canada': 'https://wiki.openfoodfacts.org/Country_Support_-_Canada',
+      'China' : 'https://wiki.openfoodfacts.org/Country_Support_-_China',
+      'Europe': 'https://wiki.openfoodfacts.org/Country_Support_-_Europe',
+      'France': 'https://wiki.openfoodfacts.org/Local_Communities/FrenchTeam/Country_Support',
+      'Germany': 'https://wiki.openfoodfacts.org/Country_Support_-_Germany',
+      'India': 'https://wiki.openfoodfacts.org/Country_Support_-_India',
+      'Italy': 'https://wiki.openfoodfacts.org/Country_Support_-_Italy',
+      'Japan': 'https://wiki.openfoodfacts.org/Country_Support_-_Japan',
+      'Mexico': 'https://wiki.openfoodfacts.org/Country_Support_-_Mexico',
+      'Netherlands': 'https://wiki.openfoodfacts.org/Country_Support_-_Netherlands',
+      'Spain': 'https://wiki.openfoodfacts.org/Country_Support_-_Spain',
+      'Russia': 'https://wiki.openfoodfacts.org/Country_Support_-_Russia',
+      'South Korea': 'https://wiki.openfoodfacts.org/Country_Support_-_South_Korea',
+      'United States': 'https://wiki.openfoodfacts.org/Country_Support_-_United_States',
+      // 'Belgium': 'https://wiki.openfoodfacts.org/Country_Support_-_Belgium',
+      // 'Bulgaria': 'https://wiki.openfoodfacts.org/Local_Communities/BulgarianTeam',
```

Comment:
Please don't put useless comments.

Reply:
I have harmonized on the first pattern FYI